src/modules/hardwareWallet/components/SelectHardwareDeviceModal/components/HwDeviceItem.js

```diff
@@ -16,17 +21,18 @@ function HwDeviceItem({ hwDevice }) {
     dispatch(setCurrentHWDevice(hwDevice));
   }
 
-  const usbPort = path
-    ?.split('/')
-    ?.find((segment) => segment.startsWith('usb'))
-    ?.split('@')[0];
+  async function handleShowPubkeyOnDevice() {
+    await getPubKey(hwDevice.path, undefined, true);
+  }
 
   return (
     <div className={styles.hwDevice}>
       <Icon name="iconLedgerDevice" className={styles.hwWalletIcon} />
       <div className={styles.infoContainer}>
         <h5 className={styles.modelInfo}>{`${manufacturer} ${product}`}</h5>
-        <span className={styles.path}>{usbPort}</span>
+        <TertiaryButton className={classNames(styles.btnShowPubkey)} onClick={handleShowPubkeyOnDevice}>
+          {t('Show pubkey on device')}
```

Comment:
I think public key option will be harder for users to view and verify, we can use the old approach.

See here 2.3 option.

![Screenshot 2023-05-08 at 1 44 56 PM](https://user-images.githubusercontent.com/6449871/236815706-dda99e33-b10d-46f7-a236-64f7ac79e284.png)